profiles/manatime/manatime.install

```diff
@@ -11,7 +11,9 @@ function manatime_install() {
   // Create a default role for site administrators,
   // with all available permissions assigned.
   $admin_role = new stdClass();
+  $admin_role->name = 'super admin';
   $admin_role->name = 'administrator';
+  $admin_role->name = 'collaborator';
```

Comment:
You need a new stdClass using a variable with a meaningful name (you can also use a loop) and a user_role_save call per each role.